object/check.go

```diff
@@ -387,7 +389,7 @@ func CheckUserPassword(organization string, username string, password string, la
 		}
 
 		// only for LDAP users
-		err = checkLdapUserPassword(user, password, lang)
+		*ldapServerId, err = checkLdapUserPassword(user, password, lang)
```

Comment:
What do you think - if instead of specifying method `CheckUserPassword` used this method `checkLdapUserPassword` separately?